### PR TITLE
Blazor integration for anchor and anchor button

### PR DESCRIPTION
**Pull request by m-akinc**
# Pull Request

## 🤨 Rationale

Standard Blazor support for new components

## 👩‍💻 Implementation

- Created Blazor wrappers for nimble-anchor and nimble-anchor-button
- Added unit tests
- Updated example program
- Updated README
- Noticed that wrapper for nimble-button was missing ContentHidden property, so added it (and added use case to example app)

## 🧪 Testing

Unit tests created, ran example app

## ✅ Checklist

- [x] I have updated the project documentation to reflect my changes or determined no changes are needed.

### CODE REVIEW

**Review by jattasNI — APPROVED** [review]

**Review by msmithNI — APPROVED** [review]

**Review by rajsite — APPROVED** [review]

File: packages/nimble-blazor/NimbleBlazor/Components/NimbleAnchor.razor.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.AspNetCore.Components;
+
+namespace NimbleBlazor;
+
+public partial class NimbleAnchor : ComponentBase
```
Comment (atmgrifter00) [inline packages/nimble-blazor/NimbleBlazor/Components/NimbleAnchor.razor.cs:5]: Can we document these properties? I don't mind if they're essentially a copy/paste of the FAST docs.

File: packages/nimble-blazor/NimbleBlazor/Components/NimbleAnchorButton.razor.cs [hunk 2]
```diff
@@ -0,0 +1,45 @@
+﻿using Microsoft.AspNetCore.Components;
+
+namespace NimbleBlazor;
+
+public partial class NimbleAnchorButton : ComponentBase
```
Comment (atmgrifter00) [inline packages/nimble-blazor/NimbleBlazor/Components/NimbleAnchorButton.razor.cs:5]: Document properties.

File: packages/nimble-blazor/Tests/NimbleBlazor.Tests/Unit/Components/NimbleAnchorTests.cs [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+        var anchor = context.RenderComponent<NimbleAnchor>();
+
+        Assert.Contains(expectedMarkup, anchor.Markup);
+    }
+
+    [Theory]
+    [InlineData(AnchorAppearance.Default, "<nimble-anchor>")]
+    [InlineData(AnchorAppearance.Prominent, "appearance=\"prominent\"")]
+    public void AnchorAppearance_AttributeIsSet(AnchorAppearance value, string expectedAttribute)
```
Comment (atmgrifter00) [inline packages/nimble-blazor/Tests/NimbleBlazor.Tests/Unit/Components/NimbleAnchorTests.cs:28]: Maybe rename `expectedAttribute` to `expectedMarkup` as the first string contains no attributes?

File: packages/nimble-blazor/Tests/NimbleBlazor.Tests/Unit/Components/NimbleAnchorButtonTests.cs [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    {
+        var button = RenderWithPropertySet(x => x.Appearance, value);
+
+        Assert.Contains(expectedAttribute, button.Markup);
+    }
+
+    [Theory]
+    [InlineData(ButtonAppearanceVariant.Default, "<nimble-anchor-button>")]
+    [InlineData(ButtonAppearanceVariant.Primary, "appearance-variant=\"primary\"")]
+    public void ButtonAppearanceVariant_AttributeIsSet(ButtonAppearanceVariant value, string expectedAttribute)
```
Comment (atmgrifter00) [inline packages/nimble-blazor/Tests/NimbleBlazor.Tests/Unit/Components/NimbleAnchorButtonTests.cs:39]: Maybe rename expectedAttribute to expectedMarkup as the first string contains no attributes?

**Review by atmgrifter00 — APPROVED** [review]

File: packages/nimble-blazor/NimbleBlazor/Components/NimbleAnchorButton.razor.cs (hunk 2 above)
Comment (m-akinc) [reply packages/nimble-blazor/NimbleBlazor/Components/NimbleAnchorButton.razor.cs:5]: Done

File: packages/nimble-blazor/Tests/NimbleBlazor.Tests/Unit/Components/NimbleAnchorTests.cs (hunk 3 above)
Comment (m-akinc) [reply packages/nimble-blazor/Tests/NimbleBlazor.Tests/Unit/Components/NimbleAnchorTests.cs]: Done

File: packages/nimble-blazor/Tests/NimbleBlazor.Tests/Unit/Components/NimbleAnchorButtonTests.cs (hunk 4 above)
Comment (m-akinc) [reply packages/nimble-blazor/Tests/NimbleBlazor.Tests/Unit/Components/NimbleAnchorButtonTests.cs]: Done

**Pull request merged**